apps/site/pages/docs/brandless.mdx

```diff
@@ -0,0 +1,79 @@
+import { LinkButton } from '@faststore/ui'
+import { ArrowRight } from '@faststore/components'
+
+<header className="hero">
+
+# Brandless
+
+This themeable structure is made of two complementary layers: a useful and assorted library of templates for the components and sections, alongside a comprehensive set of design tokens to further customize them.
+
+<LinkButton
+  size="small"
+  variant="tertiary"
+  href="/docs/customizing"
+  icon={<ArrowRight />}
+  iconPosition="right"
+>
+  Customizing your store
+</LinkButton>
+
+</header>
+
+**Structural Styles**, the first layer, has foundational styles that lay things out in common design patterns for different use cases. It also covers a component's fundamental interactions, like showing/hiding a dropdown or moving a toggle’s knob to make it active.
+
+The second, **Theme**, is where the branding lives and what truly differentiates one UI from another. Graphic elements—like typography (and type scale), colors, imagery, borders, spacing, & more—are controlled by design tokens that a developer can easily tweak to create a unique look and feel.
+
+<img
+  src="https://vtexhelp.vtexassets.com/assets/docs/src/theming-intro___1d021c8548ba2338faba6dd2995bf7ce.png"
+  align="right"
+  style={{ width: '70%', display: 'block', margin: '0 auto' }}
+/>
```

Comment:
```suggestion
<div style={{ display: 'flex', marginTop: '12px' }}>
<img
  src="https://vtexhelp.vtexassets.com/assets/docs/src/theming-intro___1d021c8548ba2338faba6dd2995bf7ce.png"
  style={{ width: '95%', display: 'block', margin: '0 auto' }}
/>
</div>
```

Reply:
I think it looks better in a single row. WTDY?

<img width="1010" alt="image" src="https://user-images.githubusercontent.com/3356699/218728600-deeb58a5-87b5-441e-8b0a-33eecad2f8a9.png">